manifests/p/PlayStation/DualSenseFWUpdater/1.0.0.2/PlayStation.DualSenseFWUpdater.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PlayStation.DualSenseFWUpdater
+PackageVersion: 1.0.0.2
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://fwupdater.dl.playstation.net/fwupdater/FWupdaterInstaller.exe
+  InstallerSha256: F558C839BA897B42AAA51E2215D50AC3BD04E3A0E008F8DD1EBAB4BA2DEE04EF
+  InstallerSwitches:
+    Silent: /s /v/qn
+    SilentWithProgress: /s /v/qb

```

```suggestion
    SilentWithProgress: /s /v/qb
  AppsAndFeaturesEntries: 
  - DisplayName: Firmware updater for DualSense™ wireless controller
    ProductCode: '{9505317F-5C12-489D-8C97-88511D9985B6}'
    UpgradeCode: '{028E5A2A-61C3-4789-8186-3C1E10BBFD89}'
    InstallerType: msi
```
Because this is an MSI installer packaged inside of an EXE, the AppsAndFeaturesEntries need to be specified in order for users to be able to upgrade in the future